contrib/kml_export.lua

```diff
@@ -263,8 +100,7 @@ local function create_kml_file(storage, image_table, extra_data)
 
     -- Creates dir if not exsists
     imageFoldername = "files"..PS
-    local mkdirCommand = "mkdir -p "..exportDirectory..PS..imageFoldername
-    dt.control.execute(mkdirCommand)
+    df.mkdir(df.sanitize_filename(xportDirectory..PS..imageFoldername))
```

Comment:
you are missing an 'e' from 'xportDirectory'

Reply:
Thanks fixed